wrangler-core/src/main/java/io/cdap/wrangler/datamodel/HTTPSchemaLoader.java

```diff
@@ -0,0 +1,168 @@
+/*
+ *  Copyright © 2020 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+package io.cdap.wrangler.datamodel;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import io.cdap.wrangler.clients.RestClientException;
+import io.cdap.wrangler.utils.AvroSchemaLoader;
+import io.cdap.wrangler.utils.Manifest;
+import org.apache.avro.Schema;
+import org.apache.commons.collections4.SetValuedMap;
+import org.apache.commons.collections4.multimap.HashSetValuedHashMap;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.function.Function;
+
+/**
+ * The {@link HTTPSchemaLoader} object loads an AVRO data model schema located in a GCS bucket.
```

Comment:
this will load schema from any url correct?

Reply:
Updated, done.